src/neuroconv/datainterfaces/behavior/deeplabcut/deeplabcutdatainterface.py

```diff
@@ -144,8 +103,9 @@ def add_to_nwbfile(
         metadata: dict
             metadata info for constructing the nwb file (optional).
         """
+        from ._dlc_utils import add_subject_to_nwbfile
 
-        write_subject_to_nwb(
+        add_subject_to_nwbfile(
```

Comment:
How are exposing the pose_estimation_kwargs here? 

Are you thinking on doing this in another PR?

Reply:
That is done in #917, the motivation for this PR

Reply:
Ah, great, I only saw the issue and forgot about the PR. Thanks.